src/infrastructures/AudioRenderer.ts

```diff
@@ -233,6 +238,38 @@ export class Transport {
   }
 }
 
+export class OfflineTransport implements BaseTransport {
```

Comment:
こちらも用途を少しだけ書くと道標になるかなと思いました！　こんな感じでしょうか（コメントにこだわりはないです･･･！）
```suggestion
/**
 * 登録されているシーケンスのイベントをスケジュールします。主に保存用途です。
 */
export class OfflineTransport implements BaseTransport {
```